src/main/java/com/arabiccalligraphy/api/DAO/ArtworkDAO.java

```diff
@@ -66,8 +66,17 @@ private void loadArtworkFromFile(String fileName) throws IOException {
             //Artist creation
             Artist artist = new Artist(contentItem.get("artist").asText());
 
+            //width creation
+            int width = contentItem.get("width").asInt();
+
+            //height creation
+            int height = contentItem.get("height").asInt();
+
+            //id creation
+            int id = contentItem.get("id").asInt();
```

Comment:
I think ids are actually alphanumeric `String`s like this:
https://github.com/deemost/calligraphy-spring/blob/main/src/main/resources/data/index.json#L4

So just use `asText()` instead here.